lib/hooks.js

```diff
@@ -1,17 +1,13 @@
 'use strict'
 
-const applicationHooks = [
-  'onGatewayReplaceSchema'
```

Comment:
I think this was used in the validation of hooks. How does the validation still work for `onGatewayReplaceSchema` with the new package?